website/docs/guides/migration/versions/02-upgrading-to-v1.5.md

```diff
@@ -40,11 +40,14 @@ Changes planned for v1.5:
 
 ### For consumers of dbt artifacts (metadata)
 
-The manifest schema version will be updated to `v9`. Specific changes to be noted here.
+The manifest schema version will be updated to `v9`. Specific changes:
+- Addition of `groups` as a top-level key
+- Addition of `access` as a top-level node config for models
+- Addition of `group` and `contract` as node configs
 
 ### For maintainers of adapter plugins
 
-Coming soon: GH discussion detailing interface changes and offering a forum for Q&A
+For more detailed information and to ask any questions, please visit [dbt-core/discussions/6624](https://github.com/dbt-labs/dbt-core/discussions/6624).
```

Comment:
Stealing from https://github.com/dbt-labs/docs.getdbt.com/pull/3052 - no reason to delay getting this info live